frontend/src/metabase/query_builder/components/data-search/SearchResultItem.jsx

```diff
@@ -0,0 +1,110 @@
+import styled from "styled-components";
+import React from "react";
+import PropTypes from "prop-types";
+
+import Icon from "metabase/components/Icon";
+import Text from "metabase/components/type/Text";
+
+import Schema from "metabase/entities/schemas";
+import Database from "metabase/entities/databases";
+
+import { color, lighten } from "metabase/lib/colors";
+
+export function SearchResultItem({ item, onSelect }) {
+  const handleClick = () => onSelect(item);
+
+  return (
+    <SearchResultItemRoot onClick={handleClick}>
+      <IconWrapper>
+        <Icon name="table" size={26} />
+      </IconWrapper>
+      <Details>
+        <Title>{item.name}</Title>
+        <Text fontSize={13} mb={0} mt={0}>
+          <ItemLocation item={item} />
+        </Text>
+      </Details>
+    </SearchResultItemRoot>
+  );
+}
+
+SearchResultItem.propTypes = {
+  item: PropTypes.object.isRequired,
+  onSelect: PropTypes.func.isRequired,
+};
+
+const Title = styled.div`
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 17px;
+`;
+
+const ItemLocation = ({ item }) => {
```

Comment:
It'd be nice if we could get these to match what we're doing in global search e.g. `Saved question in <collection>` or `Table in <database>` I'm finding that collections and tables are feeling quite similar in the results which can feel a bit confusing unless you know what's going on.

Reply:
Implemented that, please take a look:
<img width="353" alt="Screen Shot 2021-05-05 at 02 16 13" src="https://user-images.githubusercontent.com/14301985/117081097-e0c19b80-ad47-11eb-936e-448563457af9.png">